pageserver/src/tenant/timeline.rs

```diff
@@ -413,7 +413,11 @@ pub struct Timeline {
     /// Keep aux directory cache to avoid it's reconstruction on each update
     pub(crate) aux_files: tokio::sync::Mutex<AuxFilesState>,
 
+    /// Size estimator for aux file v2
     pub(crate) aux_file_size_estimator: AuxFileSizeEstimator,
+
+    /// Indicate whether aux file v2 storage is enabled.
+    pub last_aux_file_policy: AtomicRuntimeAuxFilePolicy,
```

Comment:
```suggestion
    pub(crate) last_aux_file_policy: AtomicRuntimeAuxFilePolicy,
```

Reply:
it's exposed over http API so it needs public access

Reply:
> it's exposed over http API so it needs public access

I am not sure what does that even mean? The HTTP API is in the same crate... Are you thinking of something more philosophical like all HTTP accessible fields must be `pub`? Instead we use `pub(crate)` because that's how we get dead code warnings, we cannot get those for `pub` with the current rust tooling.

Reply:
Okay... I think this is already fixed; the only place where we have `pub last_aux_file_policy: Option<AuxFilePolicy>` is in pageserver_api/src/models.rs. I might have mistaken this comment with the line in that file.